src/main/webapp/scripts/auth.js

```diff
@@ -19,11 +19,16 @@
 const PHONE_NUMBER_LENGTH = 10;
 
 // Call to Auth servlet to get user log information
-function getLogStatus() {
+/** @param {string} fileName*/
+function getLogStatus(fileName) {
   fetch('/log-status').then(response => response.json()).then((logStatus) => {
-    if(!logStatus.isRegistered && logStatus.isLogged) {
+    if (!logStatus.isRegistered && logStatus.isLogged) {
       redirectToRegistrationForm();
     }
+    
+    if (fileName === 'home') {
```

Comment:
Nit: I'd make this a constant variable and use it here.

`
const TUTORIAL_FILENAME = ...
if (fileName === TUTORIAL_FILENAME) {...}
`

Reply:
Thanks for the suggestion Aki!